source/material_model/steinberger.cc

```diff
@@ -187,13 +189,21 @@ namespace aspect
         delta_temperature = temperature-adiabatic_temperature;
 
       // For an explanation on this formula see the Steinberger & Calderwood 2006 paper
+      //The lateral variation of viscosity due to lateral temperature (Visc_lT)
+      //Visc_lT = exp [(-1)*(H/nR)*dT/(T_adiabatic*(T_adiabatic + dT)], Eq. 6 of the paper
```

Comment:
I think what @naliboff was asking for was a more detailed explanation of why these things are multiplied together. I have made a suggestion below:
```suggestion
      // We here compute the lateral variation of viscosity due to temperature (vis_lateral) as
      // V_lT = exp [-(H/nR)*dT/(T_adiabatic*(T_adiabatic + dT))] as in Eq. 6 of the paper.
      // We get H/nR from the lateral_viscosity_lookup->lateral_viscosity function. 
```

Reply:
@jdannberg Thanks a lot for your review! I made the changes.